plane/source/docs/apms-failsafe-function.rst

```diff
@@ -204,37 +243,28 @@ setting.
    <td>1</td>
    <td>ReturnToLaunch</td>
    </tr>
+   <tr>
+   <td>2</td>
+   <td>FBWA Glide</td>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>Deploy Parachute</td>
+   </tr>
    </tbody>
    </table>
 
-Failsafe battery voltage (Plane:FS_BATT_VOLTAGE)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In a QuadPlane, if in VTOL operation in modes others than AUTO or GUIDED, the action taken will be either a QRTL or QLAND, depending on the :ref:`Q_RTL_MODE<Q_RTL_MODE>` bit mask setting for bit 5. And if in fixed-wing operation, and the long or short failsafe action is a mode change to RTL, then the :ref:`Q_RTL_MODE<Q_RTL_MODE>` will determine behavior at the end of that RTL, just as in the case of a regular mode change to RTL.
 
-Battery voltage to trigger failsafe. Set to 0 to disable battery voltage
-failsafe. If the battery voltage drops below this voltage then the plane
-will RTL
+GCS failsafe enable (:ref:`FS_GCS_ENABL<FS_GCS_ENABL>` )
+--------------------------------------------------------
 
--  Units: Volts
-
-Failsafe battery milliAmpHours (Plane:FS_BATT_MAH)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Battery capacity remaining to trigger failsafe. Set to 0 to disable
-battery remaining failsafe. If the battery remaining drops below this
-level then the plane will RTL
-
--  Units: mAh
+Enable ground control station telemetry failsafe. Failsafe will trigger
+after :ref:`FS_SHORT_TIMEOUT<FS_SHORT_TIMEOUT>` and/or :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` seconds of no MAVLink heartbeat or RC Override messages.
 
-GCS failsafe enable (Plane:FS_GCS_ENABL)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. warning:: Enabling this option opens up the possibility of your plane going into failsafe mode and running the motor on the ground it it loses contact with your ground station. If this option is enabled on an electric plane then either use a separate motor arming switch or remove the propeller in any ground testing.
```

Comment:
the code will try to detect if the vehicle is flying and avoid running the motor if not flying, but like all heuristics this can fail

Reply:
added